src/scripts/main.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', (event) => {
+  const spiderWidth = spider.width;
+  const spiderHeight = spider.height;
+  const wallWidth = wall.clientWidth;
+  const wallHeigth = wall.clientHeight;
+  const positionX = event.offsetX - spiderWidth / 2;
+  const positionY = event.offsetY - spiderHeight / 2;
```

Comment:
Відступи не будуть зайвими.
```suggestion
  const spiderWidth = spider.width;
  const spiderHeight = spider.height;
  
  const wallWidth = wall.clientWidth;
  const wallHeigth = wall.clientHeight;
  
  const positionX = event.offsetX - spiderWidth / 2;
  const positionY = event.offsetY - spiderHeight / 2;
```